### ISSUE TITLE
Is this project abandoned? EDIT: no, not anymore

### ISSUE THREAD

**Issue by kfkonrad** [wagoodman/dive#553]
**What happened**:

No commit has been checked into main for 8 months. There's plenty of open issues with little to no progress and many open PRs, some with trivial changes, others with bigger features. Many of them untouched for months

**What you expected to happen**:

There's three main scenarios, which one you choose (if any at all) is of course completely up to you. I'm just laying out the options I see.

1. Someone should appoint one (or preferably multiple) new maintainers and hand dive over to them to ensure dive is revived.
2. Make it official and archive the repository. If a well-maintained fork exists I please point to it as the de-facto successor in the README. In any case please add at least the info/warning/banner about the project being archived to the README.
3. The current maintainer(s) could pick up the PRs and issues that exist and merge quality PRs or request changes where necessary. If you don't have time for implementing new features maybe reviewing PRs is still manageable

**How to reproduce it (as minimally and precisely as possible)**:

Not applicable, I'd rather not reproduce this scenario in the futre

**Anything else we need to know?**:

dive is pretty good IMO, but it still lacks a little polish and some quality of life feature. I'd hate to see it go the way of the dodo as there's no alternative TUI with similar functionality

**Comment by voltagex** [comment]
https://github.com/wagoodman/dive/issues?q=is%3Aissue+is%3Aopen+abandoned

https://github.com/wagoodman/dive/issues/388 points towards a more active fork https://github.com/jauderho/dive

**Comment by kfkonrad** [comment]
Thanks, I'm aware of that fork. There's little to no feature development going on there, mostly just dependency version bumps from what I gather. Better than nothing and definitely appreciated, but I wouldn't call that actively maintained

**Comment by black-snow** [comment]
tl;dr: yes

**Comment by n-rodriguez** [comment]
> tl;dr: yes

it's a bit sad to learn this 😢

**Comment by joschi** [comment]
I was a bit annoyed with Dive not working properly on macOS with Apple Silicon, so I created a fork and applied some of the open PRs from this repository.

See https://github.com/joschi/dive/releases/tag/v0.13.1 for the latest release.

Disclaimer: I really don't want to maintain the fork and would be much happier if @wagoodman would incorporate the changes here, in the upstream repository.

Alternatively, it could move to a dedicated GitHub organization and add more maintainers. 🤷

**Comment by mprimeaux** [comment]
@joschi I'm willing to be a maintainer as I'd like to see this project garner more traction. As to moving it to a dedicated GitHub organization, happy to discuss this as well.

**Comment by orpheusx** [comment]
@joschi thanks for forking&fixing dive. It wasn't working at all for me on my M1.

**Comment by rzr** [comment]
I see it a good candidate for @abandonware org:

https://rzr.github.io/rzr-presentations/docs/abandonware/

If there is any interest to relocate the project there, please reach me

https://github.com/abandonware/abandonware.github.io/issues/5

**Comment by gabrieljones** [comment]
> Indeed, I’m alive and well! Regarding dive, I hear your interest… give me a few weeks to see how I can make more time for you and dive in general. I don’t want to leave the project in the state it’s in, but I also am not comfortable handing over control of such a large project either. @wagoodman 2024-01-17

**Comment by stuaxocabinetoffice** [comment]
I can confirm that the changes the branch @joschi incorporated enabled me to run `dive` on an apple silicon Mac, if you're considering which PRs to incorporate.

**Comment by polarathene** [comment]
Linking for update: https://github.com/wagoodman/dive/issues/568

**Comment by wagoodman** [comment]
Closing since this is now outdated 🎉
